src/pages/Profile/components/Friends/index.tsx

```diff
@@ -40,6 +43,7 @@ const Friends: React.FC<FriendsProps> = ({ friends, setFriends }) => {
         isVisible={showDeck}
         setVisibility={setShowDeck}
       />
+      <Toast show={showToast} setShow={setShowToast} message={toastMessage} />
```

Comment:
is this line/toast setup being done everywhere like this, if so we might want to look at a better solution with 1 toast manager for the whole application. keep it as is for now but we should probably make an issue about it.